vitrina/datasets/models.py

```diff
@@ -121,6 +121,27 @@ def get_tag_list(self):
         return str(self.tags).replace(" ", "").split(',') if self.tags else []
 
 
+class DatasetMembers(models.Model):
+    CREATOR = 'dct:creator'
+    CONTRIBUTOR = 'dct:contributor'
+    PUBLISHER = 'dct:publisher'
+
+    ROLE_CHOICES = (
+        (CREATOR, _('Creator')),
+        (CONTRIBUTOR, _('Contributor')),
+        (PUBLISHER, _('Publisher')),
+    )
+    user = models.ForeignKey(User, models.DO_NOTHING, blank=True)
+    organization = models.ForeignKey(Organization, models.DO_NOTHING)
```

Comment:
```suggestion
    organization = models.ForeignKey(Organization, models.CASCADE)
```

We don't want to leave corrupted database state, when an organization is deleted, with `DO_NOTHING` this will leave a broken reference.

We also need a `dataset` otherwise this is not a dataset member, but an organization member.